From 7ca99d75b3c810546fec0418662586e34be141cd Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Wed, 5 Sep 2012 08:10:13 +0000
Subject: [PATCH] abuild: add version number to provides

So it is not treated as a special 'virtual'
We use version number 0 for now.
---
 abuild.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/abuild.in b/abuild.in
index 207cc66..f4d724c 100755
--- a/abuild.in
+++ b/abuild.in
@@ -921,7 +921,7 @@ trace_apk_deps() {
 
 	echo "# automatically detected:" >> "$dir"/.PKGINFO
 	if [ -f "$dir"/.provides-so ]; then
-		sed 's/^/provides = /' "$dir"/.provides-so >> "$dir"/.PKGINFO
+		sed 's/^\(.*\)/provides = \1=0/' "$dir"/.provides-so >> "$dir"/.PKGINFO
 	fi
 	[ -z "$autodeps" ] && return 0
 	for i in $autodeps; do
-- 
1.7.12

